helm-charts/common/tei/templates/NOTES.txt

```diff
@@ -13,7 +13,7 @@
 {{- end }}
 
 2.   Use this command to verify tei service:
-  curl ${tei_svc_ip}:6006/embed \
+  curl http://${tei_svc_ip}/embed \
```

Comment:
why do we no longer have port number?

Reply:
it's not user configurable, and happens to be 80.